README.md

```diff
@@ -11,70 +11,70 @@ Search [Crystal Shards](https://crystalshards.xyz) or follow announcements [Crys
 
 Contributions are welcome. Please take a quick look at the [contribution guidelines](https://github.com/veelenga/awesome-crystal/blob/master/.github/CONTRIBUTING.md) first.
 
-* [Awesome Crystal](#awesome-crystal)
-  * [Algorithms and Data structures](#algorithms-and-data-structures)
-  * [Api Builders](#api-builders)
-  * [Blockchain](#blockchain)
-  * [C Bindings](#c-bindings)
-  * [Caching](#caching)
-  * [CLI Builders](#cli-builders)
-  * [CLI Utils](#cli-utils)
-  * [Code Analysis and Metrics](#code-analysis-and-metrics)
-  * [Compression](#compression)
-  * [Configuration](#configuration)
-  * [Converters](#converters)
-  * [Cryptography](#cryptography)
-  * [Data Formats](#data-formats)
-  * [Data Generators](#data-generators)
-  * [Database Drivers/Clients](#database-driversclients)
-  * [Database Tools](#database-tools)
-  * [Debugging](#debugging)
-  * [Dependency Injection](#dependency-injection)
-  * [Development Tools](#development-tools)
-  * [Email](#email)
-  * [Environment Management](#environment-management)
-  * [Examples and funny stuff](#examples-and-funny-stuff)
-  * [Feature Flipping](#feature-flipping)
-  * [Framework Components](#framework-components)
-  * [Game Development](#game-development)
-  * [HTML Builders](#html-builders)
-  * [HTML/XML parsing](#htmlxml-parsing)
-  * [HTTP](#http)
-  * [Image Processing](#image-processing)
-  * [Implementations/Compilers](#implementationscompilers)
-  * [Logging and monitoring](#logging-and-monitoring)
-  * [Machine Learning](#machine-learning)
-  * [Markdown/Text Processors](#markdowntext-processors)
-  * [Misc](#misc)
-  * [Network Protocols](#network-protocols)
-  * [Networking](#networking)
-  * [ORM/ODM Extensions](#ormodm-extensions)
-  * [Package Management](#package-management)
-  * [Processes and Threads](#processes-and-threads)
-  * [Project Generators](#project-generators)
-  * [Queues and Messaging](#queues-and-messaging)
-  * [Routing](#routing)
-  * [Scheduling](#scheduling)
-  * [Science and Data analysis](#science-and-data-analysis)
-  * [Search](#search)
-  * [Serverless Computing](#serverless-computing)
-  * [System](#system)
-  * [Task management](#task-management)
-  * [Template Engine](#template-engine)
-  * [Testing](#testing)
-  * [Third-party APIs](#third-party-apis)
-  * [Validation](#validation)
-  * [Web Frameworks](#web-frameworks)
-  * [Web Servers](#web-servers)
-* [Community](#community)
-  * [Unofficial](#unofficial)
-* [Resources](#resources)
-  * [Official Documentation Translations](#official-documentation-translations)
-* [Services and Apps](#services-and-apps)
-* [Tools](#tools)
-  * [DevOps](#devops)
-  * [Editor Plugins](#editor-plugins)
-  * [Shell Plugins](#shell-plugins)
+- [Awesome Crystal](#awesome-crystal)
```

Comment:
What was wrong with `*`?

Reply:
oh sorry, seems like some formatter ran & changed something. I should have taken a look  at changes before commiting :(.

Let me change those again.

Reply:
done